addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -511,6 +511,31 @@ export function isImageSupportedForStyle(img) {
 
     return !isTFieldImg && !isEditableRootElement;
 }
+/**
+ * @param {HTMLImageElement} img
+ * @returns {Promise<Boolean>}
+ */
+export async function isImageCorsProtected(img) {
+    const src = img.getAttribute('src');
+    if (!src) {
+        return false;
+    }
+    let isCordProtected = false;
+    if (src && (!src.startsWith("/") || /\/web\/image\/\d+-redirect\//.test(src))) {
```

Comment:
No need to check "src", you already did it above?

Reply:
Whoops indeed